cmd/server/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/go-kit/kit/log"
+	"github.com/openshift/insights-operator-conditional-gathering/pkg/service"
```

Comment:
I thought it would live under github.com/RedHatInsights/

Reply:
Good point. Fixed.